compiler/haskell-ide-core/src/Development/IDE/State/RuleTypes.hs

```diff
@@ -36,44 +36,44 @@ import           Development.IDE.Types.SpanInfo
 --   Foo* means Foo for me and Foo+
 
 -- | Kick off things
-type instance RuleResult OfInterest = IdeResult ()
```

Comment:
Why rename IdeResult?

Reply:
I haven't, IdeResult is what you produce from the rules and IdeReturn is what you get back when you return a rule. Better name suggestions would be welcome